### PR TITLE
🎅 Update angular to latest

### PR DESCRIPTION
**Pull request by samccone**
The motivation here is partially to be able to confirm a memory leak in Angular core that I would like to bubble up the chain.

### CODE REVIEW

File: examples/angularjs/js/controllers/todoCtrl.js [hunk 1]
```diff
@@ -23,10 +23,9 @@ angular.module('todomvc')
 		// Monitor the current route for changes and adjust the filter accordingly.
 		$scope.$on('$routeChangeSuccess', function () {
 			var status = $scope.status = $routeParams.status || '';
-
```
Comment (samccone) [inline examples/angularjs/js/controllers/todoCtrl.js]: automated whitespace trim... I can revert or split this if you guys want

**Comment by passy** [comment]
lgtm :+1:

File: examples/angularjs/js/controllers/todoCtrl.js (hunk 1 above)
Comment (arthurvr) [inline examples/angularjs/js/controllers/todoCtrl.js]: This isn't trimming whitespace, it's removing a blank line which was there for readability so I'd prefer it to stay.

**Comment by arthurvr** [comment]
lgtm

**Pull request merged**